datus-sqlalchemy/datus_sqlalchemy/connector.py

```
@@ -0,0 +1,634 @@
+        """Execute INSERT statement."""
+        try:
+            self.connect()
+            res = self.connection.execute(text(sql))
+
+            # Get inserted primary key or row count
+            inserted_pk = None
+            try:
+                if hasattr(res, "inserted_primary_key") and res.inserted_primary_key:
+                    inserted_pk = res.inserted_primary_key
+            except Exception:
+                pass
```

Ruff (0.14.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)